.github/container/install_lingvo_aarch64.sh

```diff
@@ -27,11 +27,11 @@ pushd ${LINGVO_INSTALLED_DIR}
 # Local patches, two PR waiting to be merged + one custom patch
 git fetch origin pull/326/head:pr326
 # git fetch origin pull/328/head:pr328  ## merged upstream
-git fetch origin pull/329/head:pr329
+# git fetch origin pull/329/head:pr329
 git config user.name "JAX Toolbox"
 git config user.email "jax@toolbox"
-# git cherry-pick pr326 pr328 pr329  ## pr328 merged
-git cherry-pick pr326 pr329
+# git cherry-pick pr326 pr328 pr329  ## pr328, pr329 merged
+git cherry-pick --allow-empty pr326
```

Comment:
This looks like something we can add to [create-distribution.sh](https://github.com/NVIDIA/JAX-Toolbox/blob/distribution-keep-redundant-commits/rosetta/create-distribution.sh#L195), which would let you rewrite this section to:

```sh
cat <<EOF >/tmp/patchlist-lingvo.txt
pull/326/head
pull/328/head
pull/329/head
EOF
bash $SCRIPT_DIR/../../rosetta/create-distribution.sh \
  -d ${LINGVO_INSTALLED_DIR}
  -p /tmp/patchlist-lingvo.txt
rm -rf /tmp/patchlist-lingvo.txt
pushd ${LINGVO_INSTALLED_DIR}
```